RELEASE-NOTES.md

```diff
@@ -1,3 +1,130 @@
+# Version 23.1
+
+gem5 Version 23.1 is our first release where the development has been on GitHub.
+During this release, there have been 362 pull requests merged which comprise 416 commits with 51 unique contributors.
+
+## Significant API and user-facing changes
+
+### The gem5 build can is now configured with `kconfig`
+
+Most gem5 builds (e.g., `build/X86/gem5.opt`) are backwards compatible and require no changes to your current workflows.
```

Comment:
Most gem5 builds without customized options (excluding double dash options) (e.g. ...

Reply:
Just to confirm do you want the sentence to change to "Most gem5 builds without customized options (excluding double dash options) (e.g. , `build/X86/gem5.opt`) are backwards compatible and require no changes to your current workflows."?

Reply:
Yes